dg/lib/cloud_pak_for_data/cpd_4_0_0/cpd_manager.py

```diff
@@ -772,7 +784,7 @@ def _create_operator_subscription(
 
         if not self._openshift_manager.subscription_exists(project, subscription_metadata.name):
             if len(created_dependent_subscriptions) == 0:
-                logging.info(f"Creating subscription '{subscription_metadata.name}' for operator '{operator_name}'")
+                logging.info(f"Creating subscription '{subscription_metadata.name}' for operator '{operator_name}' from catalog '{catalog_name}'")
```

Comment:
- Remove the flake8(E501) warning by splitting the overlong string.
- In case you use Visual Studio Code, I would also suggest to format changed files with Black, which is configured in `.vscode/settings.json` (Command Palette → "Format Document" or enable `editor.formatOnSave` in settings for automatic formatting).